apps/homescreen/js/app.js

```diff
@@ -174,8 +178,7 @@ const SETTINGS_VERSION = 0;
               }
               this.bookmarks.get(id).then((bookmark) => {
                 this.addAppIcon(bookmark.data);
-                this.storeAppOrder();
```

Comment:
It's redundant - Newly added apps will always be at the end of the grid and the order is saved as soon as they get rearranged (app order being saved is backed by a marionette test, I believe).